api-doc/rapidoc.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script type="module" src="../assets/rapidoc-min.js"></script>
```

Comment:
It seems I cannot load the file locally if there is the `type="module"`. Removing it makes it load correctly : https://github.com/whatwg/html/issues/8121

Reply:
Opening it with a dedicated server solves the CORS issue